RELEASE_NOTES.md

```diff
@@ -1,5 +1,339 @@
+
+The `systemd_unit` resource now respects the `sensitive` property and does not output the content of unit files to the logs if set.
+
+### windows_firewall_rule
+
+The `windows_firewall_rule` resource has been greatly improved thanks to work by [@pschaumburg](https://github.com/pschaumburg) and [@tecracer-theinen](https://github.com/tecracer-theinen).
+
+- New `icmp_type` property, which allows setting the ICMP type when setting up ICMP protocol rules.
+- New `displayname` property, which allows defining the display name of the firewall rule.
+- New `group` property, which allows you to specify that only matching firewall rules of the indicated group association are copied.
+- The `description` property will now update if changed.
+- Fixed setting rules with multiple profiles.
+
+### windows_package
+
+The `windows_package` resource now considers `3010` to be a valid exit code by default. The `3010` exit code means that a package has successfully installed, but requires a reboot.
+
+## YAML Recipes
+
+TODO: Write some stuff
+
+## Custom Resource Improvements
+
+### Improved property require behavior
+
+https://github.com/chef/chef/pull/9688
+
+This should be linked to from the breaking change above
+
+### Resource Partials
+
+https://github.com/chef/chef/pull/9632
+
+### after_resource
+
+A new `after_resource` state has been added to resources that allows you to better control the state information reporting to Chef Automate when a resource converges.
+
+TODO: Expand this from https://github.com/chef/chef/pull/9562
+
+### identity Improvements
+
+A resource's name property is now set to be the identity property by default, and to have desired_state: false by default. This eliminates the need to set `identity: true, desired_state: false` on these properties and better exposes identity data to handler and reporting.
+
+### compile_time property
+
+The compile_time property is now defined for all custom resources so there's no need to add your own compile-time logic to your resource.
+
+## Other Improvements
+
+### Up to 25% smaller on disk
+
+We've optimized the files that ship with Chef Infra Client and eliminated many unecessary files from the installation reducing the on disk size of Chef Infra Client by up to 25%.
+
+### Windows Performance Improvements
+
+We've optimized the Chef Infra Client for modern Windows releases and improved the performance on these systems.
+
+TODO: This needs some wordsmithing
+
+### Simpler Version Comparisons with node[:platform_version]
+
+The `node['platform_version']` attribute returned from Ohai can now be intelligently compared as a version instead of a String or Integer. Previously to compare the platform_version many users would first convert the version String to a Float with `node['platform_version']`. This introduced problems on many platforms such as macOS where macOS 10.9 would appear to be a greater version number than 10.15. You can now directly compare the version without converting it first.
```

Comment:
```suggestion
The `node['platform_version']` attribute returned from Ohai can now be intelligently compared as a version instead of a String or Integer. Previously, to compare the platform_version, many users would first convert the version String to a Float with `node['platform_version']`. This introduced problems on many platforms such as macOS where macOS 10.9 would appear to be a greater version number than 10.15. You can now directly compare the version without converting it first.
```